Мухаметов Денис

denexapp opened the pull request:
[Посмотреть решение](https://rawgit.com/Denexapp/markup-task-6/master/index.html)

Commit:
Я сделал

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

Commit:
Линтинг в style.css

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

Commit:
Линтинг в style.css

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

Commit:
WebStorm удалял пробелы

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

Commit:
Попробую VS Code

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

Commit:
Забыл, что тут надо сохранять

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

Commit:
Этот пробел..

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

Commit:
Может так

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

Commit:
Убрал длинный url

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

chipolinka commented:
Привет!

- При наведении на картинки у тебя всё скачет (все картинки опускаются вниз, к переключателям). А описание поднимает картинку вверх. Оно не должно влиять на положение картинки. Пересмотри макеты снова.
- Когда вылазит модальное окно с увеличенной картинкой: если курсор находится за пределами браузера, то описание картинки исчезает, как только я возвращаю курсор, описание появляется. Кажется, что это не должно зависеть.

chipolinka commented on index.css:
```diff
@@ -0,0 +1,213 @@
+    display: none;
+}
+
+.slider:nth-child(1):checked ~ .vegetables:nth-of-type(1)
```
Можно все `display: block` объединить в одно правило

denexapp replied:
Как?

chipolinka commented:
:tomato:

Commit:
Исправил всё, проверил в ff и в я.браузере, пришлось сделать фиксированную высоту картинок
Насчёт описания при открытой картинке: так пришлось сделать из-за запрета на использование js, я посмотрел, Олег Мохов принимал другие пр где было такое-же поведение.

honest-hrundel commented:
:warning: Вы не были зарегистрированы на курс.<br>Попробуйте написать организаторам (voirina@yandex-team.ru).<br>Или приходите к нам учиться осенью 2017 года.
